src/DefaultHandle.jsx

```diff
@@ -52,10 +48,13 @@ DefaultHandle.propTypes = propTypes;
 
 DefaultHandle.defaultProps = defaultProps;
 
+const DEFAULT_HANDLE_WIDTH = 1.9 * 0.8;
```

Comment:
with theming, would we be able to define these values in the theme, so users could define their own default handle width?

Separately, where does 1.9 and 0.8 come from?